.gitignore

```diff
@@ -1,4 +1,5 @@
 /vendor/
+/cache/
 /config/local/
 .idea
```

Comment:
.idea nie powinna się znaleźć w .gitignore projektowym tylko globalnym:
https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore

to nie jest katalog projektowy.